src/main/java/task/Task.java

```diff
@@ -0,0 +1,27 @@
+package task;
+
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    // Returns status icon
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : " "); //return tick or X symbols
```

Comment:
Avoid magic string especially in this case when you're using a unicode character